### DISCUSSION TITLE
Cannot find type definition file for 'cypress'

### DISCUSSION THREAD

**Discussion by e-jigsaw** [cypress-io/cypress#22158] [Questions and Help]
I migrated 9 to 10.

```
/// <reference typess="cypress" />
```

says `Cannot find type definition file for 'cypress'`.

also in `tsconfig.json`

```
"compilerOptions": {
  "types": ["cypress"]
}
```

not works. `Entry point of type libracy 'cypress' specified in compilerOptions`.

What happen 9 between 10?

**Comment by Henrycheheng** [comment]
Hey, bro, is that solved now？i had the same problem

**Comment by e-jigsaw** [comment]
yeah, some 10.x version saved me.
